doc/source/reference/commands/io_flush.rst

```diff
@@ -279,6 +279,81 @@ isn't specified:
 .. include:: ../../example/reference/commands/io_flush/recursive_default.log
 .. io_flush
 
+Since 9.0.2, ``recursive`` behavior is extended to flush not only target object
+and child objects, but also related objects. The related objects are:
+
+* A referenced table
+* A related index column (There is source column in target TABLE_NAME)
+* A table of related index column (There is source column in target TABLE_NAME)
+
+It is useful not to forget flushing related objects.
+
+For example, ``--recursive yes`` is specified for ``TABLE_NAME``, this
+option executes equivalent to the following commands internally.
+
+- Flush table and its columns::
+
+    io_flush --target_name TABLE_NAME
+
+- Flush a referenced table::
+
+    io_flush --target_name REFERENCED_TABLE_NAME --recursive no
+
+- Flush a related index column (There is source column in TABLE_NAME)::
+
+    io_flush --target_name TABLE_NAME_OF_INDEX_COLUMN.INDEX_COLUMN
+
+- Flush a table of related index column (There is source column in TABLE_NAME)::
+
+    io_flush --target_name TABLE_NAME_OF_INDEX_COLUMN --recursive no
+
+To confirm whether all target objects are flushed correctly, you can check query log::
+
+    > io_flush --recursive "yes" --target_name "Users"
+    :000000000000000 flush[Users]
+    :000000000000000 flush[Terms]
+    :000000000000000 flush[Terms.users_name]
+    :000000000000000 flush[Users.name]
+    :000000000000000 flush[(anonymous:table:dat_key)]
+    :000000000000000 flush[(anonymous:column:var_size)]
+    :000000000000000 flush[(anonymous:table:hash_key)]
+    :000000000000000 flush[(anonymous:column:var_size)]
+    :000000000000000 flush[(DB)]
+    <000000000000000 rc=0
+
+In above example, specified not only ``Users`` table, related lexicon table ``Terms`` and
+index column ``Terms.users_name`` (data source is ``Users.name``) are also flushed.
+
+``flush[(anonymous:...)]`` and ``flush[(DB)]`` means that Groonga's internal objects
+are also flushed.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Log
+     - Description
+
+   * - ``flush[(anonymous:table:dat_key)]``
+     - The internal object names in DB are flushed. If ``GRN_DB_KEY=pat`` is set, ``TABLE_PAT_KEY`` is used instead.
+
+   * - ``flush[(anonymous:column:var_size)]``
+     - The internal object metadata (builtin types, token filter and so on) are flushed.
+
+       This is variable sized column which stores so called ``spec`` internally.
```

Comment:
What is `spec`?
Is `spec` the internal object metadata?
If so, how about explaining as below?

```
This is a variable sized column which stores the above internal object metadata.
```

Reply:
Thanks, It should be as suggested.